### ISSUE TITLE
Switching back to no specialization?

### ISSUE THREAD

**Issue by Lillecarl** [viperML/nh#39]
When specifying ```environment.etc."specialisation".text = "spec";``` How would I go about returning to no specialisation without nixos-rebuild? If i understand https://github.com/viperML/nh/blob/master/src/nixos.rs#L71 correctly it's currently impossible. A "default" argument to -s would be convenient to activate the base configuration again.

**Comment by viperML** [comment]
Yep, that seems to be uninmplemented. I didn't use the base spec so never ran into this.

How do you think the interface should look like? A new flag `--no-specialisation` or empty `--specialiaation` ?

**Comment by Lillecarl** [comment]
@viperML --no-specialisation would probably be the better option since it's explicit. Although I don't really have a preference 😄

**Comment by viperML** [comment]
I used to have a modified activation script that would set the default specialisation to boot. But it was a pain to maintain, so I don't actually use specialisations now.

If you are interested I could dig up my dotfiles.

**Comment by Lillecarl** [comment]
@viperML No need, thanks. I'm trying to fit NixOS into the deploy pipeline we're using at work (Packer + Launch Configs) and it was just a thought. In the end we'll just not activate our configurations until reboot to keep the machine clean from state until it starts the first time.
